.github/CONTRIBUTING.md

```diff
@@ -39,6 +39,82 @@ With that all in place, you're ready to start contributing to Carbon!
 
 ## Start contributing
 
+
+## Setting up the repository:
+## Windows guide:
+This repository runs best in a Linux envrionment, to avoid running into any errors as a windows user
+you will need to run the repo in a Linux environment using the Linux Subsystem for Windows.
+
+System requirements:
+https://docs.microsoft.com/en-us/windows/wsl/install-win10
+
+prerequisites:
+- aforementioned prerequisites
+- Windows Subsystem for Linux
+- linux operating system of your choice.
+
+## 1: Open a powershell session and enter:
```

Comment:
```suggestion
## 1: Open a PowerShell session and enter:
```

Reply:
Sorry about the grammar, I whipped this up in a rush.